db/migrate/20160310044144_add_doc_id_to_topic.rb

```diff
@@ -0,0 +1,26 @@
+class AddDocIdToTopic < ActiveRecord::Migration
+
+  def up
+    add_column :topics, :doc_id, :integer, default: 0
+    add_column :docs, :topics_count, :integer, default: 0
+    add_column :docs, :allow_comments, :boolean, default: true
+
+    if Forum.where(name: 'Doc comments').first.nil?
```

Comment:
Is there a reason the migration is adding a (seemingly random) Forum? I'm conscious that this will (obviously) run when anyone updates their Helpy instance; do they _need_ the 'Doc comments' forum? And what if they're not English? Could this be moved to `seeds.rb` or `populate.rake`?